lib/rubygems/executor.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+require 'open-uri'
+require 'json'
+require 'rubygems/platform'
+
+module Gem
+  module Web
+    class Executor
+
+      def open_page(gem, options)
+        begin
+          spec = Gem::Specification.find_by_name(gem)
+        rescue Gem::MissingSpecError => e
+          spec = fetch_remote_spec(gem)
+
+          if spec.nil?
+            puts e.message
+            puts "Could not find '#{gem}' in rubygems.org too."
```

Comment:
Maybe don't print the `e.message` ? 

change to `Could not find '#{gem}' in rubygems.org`

Reply:
Ok!